UnitySDK/Assets/ML-Agents/Scripts/Utilities.cs

```diff
@@ -28,31 +28,67 @@ public static class Utilities
             var numTextures = textures.Count;
             var width = textures[0].width;
             var height = textures[0].height;
-            var data = tensorProxy.data;
```

Comment:
Old code wrote all the Textures to the tensor at once. Now we'll do it one texture at a time.